src/app/story/storyrow/storyrow.component.css

```diff
@@ -162,3 +162,12 @@
     /* Hides the shadow effect of node's img while it is being bordered */
     opacity: 0;
 }
+
+.total-time-column {
+    position: relative;
+}
+
+.total-times {
```

Comment:
Could you please put these classes in the "Elements" section of this file? This class should be called something like `.total-time-column__total-times` to match the BEM convention.